README.md

```diff
@@ -3,11 +3,19 @@
 ## Quick Start
 
 1. Clone this repo to your local environment.
+1. Make sure you have Ruby installed by running `ruby -v`
+1. Run `bundle install` to install required gems. If you don't have Bundler installed, you can install it with `gem install bundler`
 
-2. Run `bundle install` to install required gems. If you don't have Bundler installed, you can install it with `gem install bundler`
+## Command Line Usage
 
-3. To start your server, you can run the `rackup` command. You can also start it by running `bin/start-localhost-server.sh`, but be sure to make that file executable. `chmod -x bin/start-localhost-server.sh`
+Run `./bin.convert` passing in the roman numeral you would like to convert as an argument, like so:
```

Comment:
The command here should be `./bin/convert-roman`. I like the shorter `./bin/convert` though, so maybe we want to consider changing the filename to simply `convert`?

Reply:
Wait, did the executable name already get changed to `convert`? I saw that it did in one of the commits, but when I fetched this down to my machine, it was still `convert-roman`.

Reply:
Good catch! Now that I'm thinking about it, I think `convert-roman XXX` is more intuitive than `convert XXX`, in the sense that at some point we might want to `convert-arabic`. Thoughts?

Reply:
I decided to keep it as `convert-roman` for now, for the reasons mentioned above. Let me know if you are ok with it.

Reply:
Totally okay with it, and I agree with the points you made above